Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -93,10 +93,12 @@ GCPad::GCPad(const unsigned int index) : m_index(index)
 
   // options
   groups.emplace_back(m_options = new ControllerEmu::ControlGroup(_trans("Options")));
-  m_options->AddSetting(&m_always_connected_setting,
-                        // i18n: Treat a controller as always being connected regardless of what
-                        // devices the user actually has plugged in
```

Comment:
This `i18n` comment still makes sense IMO, any particular reason why you removed it? Translators only see the string "Always Connected" and this comment makes sure they also get some context on what it _means_.

Reply:
So what do I need to do? Copy and paste the tooltip I added:
`If this is false and you have a default device,\nit will link the emulated controller connection state\nto the real device connection state.`
as `i18n`?

Reply:
I think the tooltip is a little hard to understand. But if it wasn't for that, then either putting a copy of the tooltip as an i18n comment or keeping the existing i18n comment would be fine.

Reply:
Re-written the tooltip, put back the `i18n`